Update README.md for review (#38)
desarrolladorSLP/teckers-android-app: README.md

# Teckers Android App

## About us and the project
### What is teckers?

Teckers is a school dedicated to girls and boys between nine and nineteen years old, to help them develop and strengthen the skills and knowledge necessary to create technological solutions to social problems in their community.

### What do we want do?

In this project we do not give the task of creating a functional mobile application for the android system.

> The android app of "teckes" plans to facilitate the handling and communication between students and teckers in a faster and more precise way.

Reviewer: ```suggestion
The android app of "teckers" plans to facilitate the handling and communication between students and teckers in a faster and more precise way.
```

## Technology

> It will work giving different types of tools, such as messaging and attendance control management in students.

Reviewer: ```suggestion
The app will contain different tools, such as messaging and attendance control management for students.
```

We work with the version 1.0 (the first version) of the app, currently under development.

### Development

We work with different development technologies that you must acquire for the correct operation of the application, which are the following:
<br>

| Technology        | Version           | Type  |
| ----------------- |:-----------------:| -----:|
| Android Studio    |        3.2.2      |  IDE  |

> | Kotlin            |  1.3.30           |   Plugging to an programming language |

Reviewer: ```suggestion
| Kotlin            |  1.3.30           |   Programming language |
```

<br>
<br>
If you do not have the Android Studio IDE installed you can follow these steps:

##### Windows
1. Go to [https://developer.android.com/studio] and download the installation file. 
2. If you download `.exe` file(recommended), double click on it to start it.
3. If you downloaded a `.zip` file, extract it and copy the android-studio folder to your Program Files folder. Next, open the android-studio> bin folder and then start `studio64.exe`(for 64-bit machines) or `studio.exe` (for 32-bit machines).
4. Follow the steps in the setup wizard in Android Studio and make sure you install the recommended SDK packages.
<br>
In the following video, each step of the setup procedure is shown when using the recommended `.exe` download.
[https://youtu.be/1-k5KMj8IJQ]

##### MAC
1. Go to [https://developer.android.com/studio] and download the installation file. 
2. Run the DMG file from Android Studio.
3. Drag and drop Android Studio into the "Applications" folder and then launch it.
4. Select if you want to import previous configuration options from Android Studio, and then click OK.

> 5. El asistente de configuración de Android Studio te guiará por el resto de la configuración, lo que incluye la descarga de los componentes del SDK de Android que se necesiten para el desarrollo.

Reviewer: Translate this

2. Extract the `.zip` file you downloaded in a location suitable for your applications; for example, within `/usr/local/` for your user profile or `/opt/` for shared users.
3. To start Android Studio, open a terminal, navigate to the android-studio `/bin/` directory and run `studio.sh`.
4. Select whether you want to import previous configuration options from Android Studio or not, and then click OK.
5. The Android Studio setup wizard will guide you through the rest of the setup, which includes downloading the Android SDK components that are needed for development.
<br>
The following video shows each step of the recommended configuration procedure.
[https://youtu.be/7YNaxsUPwZc]

#### Kotlin
Kotlin is a cross-platform, statically typed, general-purpose programming language with type inference. Kotlin is designed to interoperate fully with Java, and the JVM version of its standard library depends on the Java Class Library, but type inference allows its syntax to be more concise. Kotlin mainly targets the JVM, but also compiles to JavaScript or native code (via LLVM). Kotlin is sponsored by JetBrains and Google through the Kotlin Foundation.
<br>
<br>

> From the android studio application you can download kotlin like this:

Reviewer: ```suggestion
From Android Studio you can enable Kotlin like this:
```
